src/validators/FarcasterLargerAnonymitySetVerifyBody.ts

```diff
@@ -0,0 +1,8 @@
+import { IsEthereumAddress } from 'amala'
+
+export default class {
+  @IsEthereumAddress()
+  address!: string
```

Comment:
You shouldn't need this field. Think of this data to be public. If you specify the address publicly, what's the anonymity here?